packages/web-server/src/server.ts

```diff
@@ -39,6 +47,24 @@ async function serve() {
   const port = options.port ? parseInt(options.port) : redwoodConfig.web.port
   const apiUrl = redwoodConfig.web.apiUrl
 
+  if (!isFullyQualifiedUrl(apiUrl)) {
+    console.error(
+      `${chalk.red('Error')}: If you don't provide ${chalk.magenta(
+        'apiHost'
```

Comment:
Shouldn't you check if `apiHost` is provided or not before printing this error message?